packages/Main/src/Layer/TiledGeometryLayer.js

```diff
@@ -422,39 +415,16 @@ class TiledGeometryLayer extends GeometryLayer {
      * @param {Object} context - The context of the update; see the {@link
      * MainLoop} for more informations.
      * @param {TileMesh} node - The node to subdivide.
-     * @return {Promise}  { description_of_the_return_value }
      */
     subdivideNode(context, node) {
-        if (!node.pendingSubdivision && !node.children.some(n => n.layer == this)) {
+        if (!node.children.some(n => n.layer == this)) {
```

Comment:
may be could be write  
`if (node.children.length)`Because the children can only be on the same node's layer (to be verified).